srcCpp/RTIDDSImpl.cxx

```diff
@@ -172,45 +172,18 @@ void RTIDDSImpl<T>::Shutdown()
     }
 }
 
-/*********************************************************
-
- * Validate and manage the parameters
- */
 template <typename T>
-bool RTIDDSImpl<T>::validate_input()
+bool RTIDDSImpl<T>::data_size_related_calculations()
```

Comment:
Some checks need to be done after we gather the information about the participant_qos, and not before